tools/cxbe/Main.cpp

```diff
@@ -113,10 +113,11 @@ int main(int argc, char *argv[])
             }
             else if(strcmp(szOptionU, "TITLE") == 0)
             {
-                if(dwParamSize > 256)
-                    printf("WARNING: Title too long, using default title\n");
-                else
-                    strcpy(szXbeTitle, szParam);
```

Comment:
I fail to see where this strcpy ended up instead. Was it extraneous to begin with?

Reply:
This was a bug; I have fixed it.

I had originally written this PR on top of a larger refactor (which touches the argument-parser); that refactored code read the argument directly into `szXbeTitle`.

I have added the `strcpy` back in.

I've also moved the trimming out of the argument-parser. This will help with the mentioned argument-parser refactor and also deal with a long default title (in case someone ever changes it).